modules/SpyUtils.js

```diff
@@ -27,7 +27,11 @@ export const createSpy = (fn, restore = noop) => {
 
   let targetFn, thrownValue, returnValue
 
-  const spy = function spy() {
+  const spy = new Function('spy', `return function(${ // eslint-disable-line no-new-func
```

Comment:
http://kangax.github.io/compat-table/es6/ lists it as Edge, Firefox, Chrome, and Webkit.